src/app/features/animals/models/animal.ts

```diff
@@ -0,0 +1,5 @@
+export interface Animal {
+  name: string;
+  type: string;
```

Comment:
V interface `Animal` chýba premenná `imageUrl`, takže by si ju mal doplniť a nezabudni, že niektoré zvieratá túto premennú nemusia mať. Okrem toho by si mal namiesto `string` ako typ pre premennú `type` použiť enum, ktorý vyššie odporučil @marek-dano 

```suggestion
  type: AnimalType;
  imageUrl: undefined | null | string;
```